crates/hue/src/effect_duration.rs

```diff
@@ -0,0 +1,78 @@
+use crate::{error::HueResult};
+
+#[derive(PartialEq, Eq, Debug)]
```

Comment:
Please derive `Clone` and `Copy`

(`Clone` for all hue api structs, and `Copy` for the ones that are "free")